service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/EhrExtractStatusService.java

```diff
@@ -153,7 +156,7 @@ public boolean hasLastUpdateExceededEightDays(EhrExtractStatus ehrExtractStatus,
 
         long daysSinceLastUpdate = Duration.between(ehrExtractStatus.getEhrExtractCorePending().getSentAt(), time).toDays();
 
-        return daysSinceLastUpdate > EHR_EXTRACT_SENT_DAYS_LIMIT;
+        return daysSinceLastUpdate > ehrExtractSentDaysLimit;
```

Comment:
This is part of a method called "hasLastUpdateExceededEightDays"... it would seem to make sense for that method name to change too, if we're making the value variable.

Reply:
oh, yes, good catch @stevenmccullaghmadetech